infra/multichain-dev/main.tf

```diff
@@ -58,9 +65,22 @@ module "gce-container" {
       {
         name  = "MPC_ENV",
         value = var.env
+      },
+      {
+        name = "MPC_REDIS_URL",
+        value = var.redis_url
       }
     ])
   }
+
+  volumes = [
+      {
+        name = "host-path"
+        hostPath = {
+          path = "/var/redis"
```

Comment:
had to change this from /etc to /var. In container optimized OS, /etc is a stateless directory and ships with a default configuration. /var is persistent and gets mounted to /mnt/stateful-partition